cmd/insprd/memory/tree/brokers_test.go

```diff
@@ -0,0 +1,227 @@
+package tree
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/inspr/inspr/cmd/insprd/memory"
+	"github.com/inspr/inspr/pkg/meta/brokers"
+	metautils "github.com/inspr/inspr/pkg/meta/utils"
+	"github.com/inspr/inspr/pkg/utils"
+)
+
+func TestMemoryManager_Brokers(t *testing.T) {
+
+	tests := []struct {
+		name string
+		want memory.BrokerInterface
+	}{
+		{
+			name: "standard Brokers() behaviour",
+			want: &BrokersMemoryManager{},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			resetSingleton()
+			tmm := &MemoryManager{}
```

Comment:
`tmm` as the abreviation of `MemoryManager` leaves the confusion about what the `t` stands for

Reply:
It stands for "Tree Memory Manager"